packages/esm-patient-attachments-app/src/attachments/image-preview.component.tsx

```diff
@@ -37,14 +38,17 @@ export default function ImagePreview(props: ImagePreviewProps) {
         alt={t('webcamPreview', 'Webcam preview')}
       />
       {props.collectCaption && (
-        <input
-          type="text"
-          placeholder={t('attachmentCaptionInstruction', 'Enter a caption for the image')}
-          onChange={updateCaption}
-        />
+        <div style={{ marginBottom: '.3rem' }}>
+          <TextInput
+            id="caption"
+            labelText={null}
+            placeholder={t('attachmentCaptionInstruction', 'Enter caption')}
+            onChange={updateCaption}
+          />
+        </div>
       )}
       <UserHasAccess privilege="Create Attachment">
-        <ButtonSet style={{ width: '50%' }}>
+        <ButtonSet style={{ width: '50%' }} className={styles.buttonSetOverrides}>
```

Comment:
The `width: 50%` declaration could just be added to the `buttonSetOverrides` rule.